builtin/v9/migration/miner.go

```diff
@@ -151,10 +165,10 @@ func (m minerMigrator) migrateState(ctx context.Context, store cbor.IpldStore, i
 		PreCommittedSectors:        newPrecommits,
 		PreCommittedSectorsCleanUp: inState.PreCommittedSectorsCleanUp,
 		AllocatedSectors:           inState.AllocatedSectors,
-		Sectors:                    inState.Sectors,
+		Sectors:                    newSectors,
```

Comment:
nit: lets use consistent naming between newSectors and deadlinesOut

Reply:
yeeee